specification/portal/resource-manager/Microsoft.Portal/preview/2020-09-01-preview/portal.json

```diff
@@ -500,7 +500,8 @@
                       "description": "The subtitle of the markdown part."
                     },
                     "markdownSource": {
-                      "type": "object",
+                      "type": "integer",
```

Comment:
>integer [](start = 31, length = 7)

Narrowing a type is a breaking change. Can you explain?

Reply:
We discussed internally over email:
![image](https://user-images.githubusercontent.com/22460039/94983533-766a2380-04f8-11eb-9fc3-18c1d5245399.png)

Based on this, I don't think the breaking change approval is necessary.